src/api/checks.rs

```diff
@@ -66,6 +75,28 @@ impl<'octo, 'r> CreateCheckRunBuilder<'octo, 'r> {
         self
     }
 
+    /// The final conclusion of the check.
+    /// Can be one of `success`, `failure`, `neutral`, `cancelled`, `timed_out`,
+    /// `skipped`, `stale` or `action_required`.
+    pub fn conclusion(mut self, conclusion: impl Into<String>) -> Self {
```

Comment:
Can you make this an enum with a `Display` implementation? That would make it so you can't provide an incorrect variant.

Reply:
I added the enum type, and made a few other things strongly typed as well.
The Display implementation puzzled me a bit, since most enums don't seem to have this. (Though some have a Debug derive)
Though if you still want me to add them for the enum I've created feel free to let me know.

Reply:
Serialising it is also fine, Display is just quicker and faster, because you know you're converting to a string.